source/Nevermore.IntegrationTests/SetUp/IntegrationTestDatabase.cs

```diff
@@ -14,7 +14,7 @@ public IntegrationTestDatabase()
             var username = Environment.GetEnvironmentVariable("NevermoreTestUsername");
             var password = Environment.GetEnvironmentVariable("NevermoreTestPassword");
             testDatabaseName = Environment.GetEnvironmentVariable("NevermoreTestDatabase") ?? "Nevermore-IntegrationTests";
-            var builder = new SqlConnectionStringBuilder($"Server={sqlInstance};Database={testDatabaseName};{(username == null ? "Trusted_connection=true;" : string.Empty)}")
+            var builder = new SqlConnectionStringBuilder($"Server={sqlInstance};Database={testDatabaseName};{(username == null ? "Trusted_connection=true;" : string.Empty)}; Encrypt=False;")
```

Comment:
Bumping the SqlClient altered the default for encryption. Most devs probably don't have this enabled on their dev SQL Server instance, so we can set this to avoid forcing folks to have to set another env var, or change their server settings.

Reply:
Will we need to add this on all connection strings that we have on Octopus?

Reply:
Yeah possibly / probably 👍 - good call